content/hardware/required/autopilot/index.md

```diff
@@ -0,0 +1,53 @@
++++
+title = "Autopilot (Flight Controller) Board"
+description = "Processes the pilot input and sensor data, and controls the lights and actuators on the vehicle."
+date = 2022-10-11T17:33:19+11:00
+template = "docs/page.html"
+sort_by = "weight"
+weight = 0
+draft = false
+[extra]
+lead = ""
+toc = true
+top = false
++++
+
+In the ArduSub control system, the autopilot board (also known as a flight controller) is the component which the ArduSub firmware is loaded onto. The autopilot processes the pilot input and sensor data, and controls the motors, lights, servos, and relays on the vehicle.
+
+Although there are many different autopilot boards available, here are the general characteristics:
+* Ability to load any ArduPilot binary firmware file (Copter, Plane, Rover/Boat, Sub)
+* Manual control and semi-autonomous/stablilised "flight" modes
+* Various input and output connections for connecting multiple peripherals
+* Embedded IMU(s), magnetic compass(es) and gyroscope(s) to determine vehicle's orientation
+* Capable of transmitting telemetry data, and saving vehicle logs
+
+## Recommended Autopilots
+
+The first autopilot board recommended for ArduSub was the [Pixhawk 1](https://docs.px4.io/v1.10/en/flight_controller/pixhawk.html) manufactured by 3D Robotics (3DR) from 2013-2016. When 3DR exited manufacturing hardware in 2016, different manufacturers took the open source design files and created their own flight controllers. The original Pixhawk design has been retired, and Blue Robotics has designed the Navigator as a modern replacement that is tailored to ArduSub use cases.
+
+<img src="navigator.png" alt="Navigator Flight Controller Board" width=300> <img src="pixhawk.png" alt="Pixhawk Flight Controller Board" width=300>
+
+The following list is the recommended autopilot boards for use with ArduSub:
+1. [Blue Robotics Navigator](https://bluerobotics.com/store/comm-control-power/control/navigator/) (requires Raspberry Pi 4B [Onboard Computer](../onboard-computer/))
+1. The original [3DR (3D Robotics) Pixhawk 1](https://docs.px4.io/v1.10/en/flight_controller/pixhawk.html)
+1. [mRobotics (Mayan Robotics) Pixhawk 1](https://docs.px4.io/v1.10/en/flight_controller/mro_pixhawk.html)
+1. [HolyBro Pixhawk 4](http://www.holybro.com/product/pixhawk-4/)
+
+There is a comparison [in the Blue Robotics Technical Reference](https://bluerobotics.com/learn/technical-reference#flight-controller-boards).
+
+## Not Recommended Autopilots
+
+With feedback from other users, the following autopilot boards are not recommended for use with Ardusub:
+
+1. [Cube Module](https://docs.cubepilot.org/user-guides/autopilot/the-cube-module-overview) (all colors) and corresponding carrier board
```

Comment:
what about blueos?

Reply:
Hmm, not sure what the history is for this being not recommended for ArduSub. If the issue is Companion not detecting it then the BlueOS support for Cube Orange may fix it, but if it's an ArduSub features issue or something then the BlueOS support is unrelated. Guess this needs to be looked into.